README.rst

```diff
@@ -6,7 +6,7 @@ Wand_
 =====
 
 Wand is a ``ctypes``-based simple ImageMagick_ binding for Python,
-supporting 2.6, 2.7, 3.2--3.5, and PyPy. Currently, not all
+supporting 2.6, 2.7, 3.3--3.5, and PyPy. Currently, not all
```

Comment:
It would be better if we replace `--3.5` with just a plus sign: “2.6, 2.7, 3.3+”.